### DISCUSSION TITLE
How to override inbuilt FCM library to use the latest one i downloaded in Library Manager

### DISCUSSION THREAD

**Discussion by InfiSoftApps** [Sketchware-Pro/Sketchware-Pro#875] [Q&A]
How to override inbuilt FCM library / inbuilt OneSignal library to use the latest one i downloaded in Library Manager

**Comment by j-px** [comment]
Go to Library Manager > Exclude built-in libraries and select built-in libraries that you want to override / replace. You probably want to select `OneSignal-3.14.0` with package name `com.onesignal` if you want to use a different OneSignal version, and for FCM, at least `firebase-messaging-19.0.0`, but maybe more for its dependencies.
